erts/emulator/test/jit_SUITE.erl

```diff
@@ -180,6 +180,48 @@ annotate(Config) ->
                     [Symbol, Anno])
     end.
 
+run_jmsingle_test(Param, ExpectSuccess, ErrorMsg) ->
+    Cmd = "erl +JMsingle " ++ Param ++ " -noshell " ++
+        "-eval 'io:format(\"All is well~n\"),erlang:halt(0).'",
```

Comment:
This will most likely not work on Windows. As far as I know, single quotes are not supported on Windows. The following is more likely to work:

```suggestion
        "-eval erlang:display(all_is_well),erlang:halt(0).",
```

Reply:
I have pushed a fixup for this, I will wait for a clarification from @garazdawi before dealing with the netbsd issue.